src/content/docs/mobile-monitoring/new-relic-mobile-unity/monitor-your-unity-application.mdx

```diff
@@ -0,0 +1,310 @@
+
+```groovy
+   allprojects {
+    buildscript {
+        repositories {**ARTIFACTORYREPOSITORY**
+            google()
+            jcenter()
+            mavenCentral()
+        }
+    }
+```
+
+2. Add the New Relic `classpath` to your project-level `baseProjectTemplate.gradle` file. You can usually find this file located in the Android folder in your Unity project.
+
+```groovy
+        dependencies {
+            // If you are changing the Android Gradle Plugin version, make sure it is compatible with the Gradle version preinstalled with Unity.
+            // See which Gradle version is preinstalled with Unity here: https://docs.unity3d.com/Manual/android-gradle-overview.html
+            // See the official Gradle and Android Gradle Plugin compatibility table here: https://developer.android.com/studio/releases/gradle-plugin#updating-gradle
+            // To specify a custom Gradle version in Unity, go to "Preferences > External Tools", uncheck "Gradle Installed with Unity (recommended)", and specify a path to a custom Gradle version.
+            classpath 'com.newrelic.agent.android:agent-gradle-plugin:7.2.0'
+            **BUILD_SCRIPT_DEPS**
+        }
+    }
+
+ ```
+ If you are utilizing an older version of Unity Studio, you can incorporate a lower version of the classpath as a dependency which supports Gradle version 7.0 and lower.
+
+```groovy
+      dependencies {
+          // If you are changing the Android Gradle Plugin version, make sure it is compatible with the Gradle version preinstalled with Unity.
+          // See which Gradle version is preinstalled with Unity here: https://docs.unity3d.com/Manual/android-gradle-overview.html
+          // See official Gradle and Android Gradle Plugin compatibility table here: https://developer.android.com/studio/releases/gradle-plugin#updating-gradle
+          // To specify a custom Gradle version in Unity, go to "Preferences > External Tools", uncheck "Gradle Installed with Unity (recommended)", and specify a path to a custom Gradle version.
+          classpath 'com.newrelic.agent.android:agent-gradle-plugin:6.11.0'
+          **BUILD_SCRIPT_DEPS**
+      }
+```
+
+3. Apply the `newrelic` plugin to the top of your `launcherTemplate.gradle` file. This file is typically located in the Android folder of your Unity project.
+
+```groovy
+apply plugin: "com.android.application"
+apply plugin: 'newrelic' // <-- add this
+```
+
+4. Add `INTERNET` and `ACCESS_NETWORK_STATE` permissions in your `AndroidManifest.xml` file:
+
+```xml
+<uses-permission android:name="android.permission.INTERNET" />
+<uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
+```
+</Step>
+
+<Step>
+### (iOS-only) Manage dependencies [#manage-dependencies]
+
+For iOS, check your External Dependency Manager settings:
+
+1. In your Unity IDE, go to **Assets** and open the **External Dependency Manager**.
+2. Click **iOS Resolver**.
+3. Settings: [#configure-ios]
```

Comment:
@ndesai-newrelic Is this supposed to link to content somewhere?

Reply:
@ally-sassman no.there is no link for that

Reply:
What's the [#configure-ios] for then?

Reply:
i think it is not needed.